doc/architecture/decisions/0002-git-commit-messages.md

```diff
@@ -0,0 +1,45 @@
+# 2. git commit messages
+
+Date: 2019-11-13
+
+## Status
+
+Draft
+
+## Context
+
+We need to improve the quality of the git commit messages so developers can easily understand the context of the proposed changes, throughout a concise and consistent message in order to avoid lengthy code reviews and project long term maintainability.
+
+Also, a good and structured message will support the automation of the releases notes or at least reduce the burn of it.
+
+## Decision
+
+The first interation will rely on generic rules described [here](https://chris.beams.io/posts/git-commit/), along with some specificities (e.g. GitHub issue if associated with one).
+
+Most wanted rules:
+
+1. Separate subject from body with a blank line
+2. Limit the subject line to 50 characters
+3. Capitalize the subject line
+4. Wrap the body at 72 characters
+5. Use the body to explain what and why vs. how
+
+Other rules are also important!
+
+In order to automate the release notes/changelog process, we also need to define basic semantics:
+
+| Commit message | Release type |
```

Comment:
This table is weird to read in plain text. Could you align the borders with spaces, please?